assets/src/dashboard/theme.js

```diff
@@ -176,11 +178,11 @@ const theme = {
     presets: {
       xxl: {
         family: themeFonts.primary,
-        size: 32,
+        size: 36,
         minSize: 18,
-        lineHeight: 53,
-        minLineHeight: 43,
-        letterSpacing: -0.005,
+        lineHeight: 40,
+        minLineHeight: 40,
+        letterSpacing: -0.01,
         minLetterSpacing: -0.01,
```

Comment:
this size isn't being used anywhere else in the dashboard. It's the default for the "view" heading (My Stories, Explore Templates) however that's getting completely overridden for the squish functionality so there's nothing to test on this theme update.